lib/Terminal/Print/Grid.pm6

```diff
@@ -8,7 +8,7 @@ unit monitor Terminal::Print::Grid;
 #| Internal (immutable) class holding all position-independent information about a single grid cell
 my class Cell {
     use Terminal::ANSIColor; # lexical imports FTW
-    has $.char is required;
+    has Str:D $.char is required where *.chars == 1;
```

Comment:
Have you checked the performance of this change?  Cell is pretty bare bones because it needs to be really fast ....

Reply:
There is an impact, no doubt. But it's a trade-off. And the impact itself is rather insignificant, based on `mandelbrot-pixels.p6` run:

- withoug constrain:
   ```
   16 zooms of 11676 pixels each in 1.875 seconds = 99635 pixels/second
   ```

- with constrain:
   ```
   16 zooms of 11676 pixels each in 1.905 seconds = 98047 pixels/second
   ```

The point of adding this was to prevent weird behavior when I was passing a mutli-char string instead of a single char. That was before I got better understanding of how things works inside. For now I wouldn't need this, but for somebody it could spare some time on fixing a bug.

The reason of having it here is because it's the single point of failure. Perhaps I overreacted and it'd be better to constrain parameters of `change-cell` and `print-cell`. Though it would still be a good thing to catch possible bugs.

Reply:
Updated. Sometimes the code with constrain turned on displays even better results than the code without constrain. Either there is not so many writes directly into a cell or Raku optimization kicks in.